micros/auth/config/index.js

```diff
@@ -22,4 +22,6 @@ exports.appConfig = {
   GOOGLE_CLIENT_SECRET: process.env.GOOGLE_CLIENT_SECRET,
   GOOGLE_CALLBACK_URL: process.env.GOOGLE_CALLBACK_URL,
   SESSION_SECRET_KEY: process.env.SESSION_SECRET_KEY,
+  HMAC_SECRET_KEY: process.env.HMAC_SECRET_KEY,
+  HMAC_HEADER_NAME: process.env.HMAC_HEADER_NAME,
```

Comment:
Please use `HMAC_HEADER_NAME` in your code instead of hardcoded  `X-Cloud-Signature`